torchmetrics/functional/text/rouge.py

```diff
@@ -199,24 +202,50 @@ def _rouge_score_update(
             {'fmeasure': tensor(1.), 'precision': tensor(1.), 'recall': tensor(1.)}]}
     """
     results: Dict[Union[int, str], List[Dict[str, Tensor]]] = {rouge_key: [] for rouge_key in rouge_keys_values}
+
     for pred_raw, target_raw in zip(preds, targets):
-        pred = _normalize_and_tokenize_text(pred_raw, stemmer)
-        target = _normalize_and_tokenize_text(target_raw, stemmer)
+        result_inner: Dict[Union[int, str], Dict[str, Tensor]] = {rouge_key: {} for rouge_key in rouge_keys_values}
+        list_results = []
+        for pred_raw_inner, target_raw_inner in zip(repeat(pred_raw), target_raw):
```

Comment:
```suggestion
        for target_raw_inner in target_raw:
```
It is sufficient to iterate only over the reference list.